quantifiers_and_equality.md

```diff
@@ -477,40 +477,30 @@ of the `Trans` type class. Type classes are introduced later, but the following
 small example demonstrates how to extend the `calc` notation using new `Trans` instances.
 
 ```lean
-def divides (x y : Nat) : Prop :=
+def Divides (x y : Nat) : Prop :=
   ∃ k, k*x = y
 
-def divides_trans (h₁ : divides x y) (h₂ : divides y z) : divides x z :=
+def Divides_trans (h₁ : Divides x y) (h₂ : Divides y z) : Divides x z :=
   let ⟨k₁, d₁⟩ := h₁
   let ⟨k₂, d₂⟩ := h₂
   ⟨k₁ * k₂, by rw [Nat.mul_comm k₁ k₂, Nat.mul_assoc, d₁, d₂]⟩
 
-def divides_mul (x : Nat) (k : Nat) : divides x (k*x) :=
+def Divides_mul (x : Nat) (k : Nat) : Divides x (k*x) :=
   ⟨k, rfl⟩
 
-instance : Trans divides divides divides where
-  trans := divides_trans
+instance : Trans Divides Divides Divides where
+  trans := Divides_trans
 
-example (h₁ : divides x y) (h₂ : y = z) : divides x (2*z) :=
+example (h₁ : Divides x y) (h₂ : y = z) : Divides x (2*z) :=
   calc
-    divides x y     := h₁
+    Divides x y     := h₁
     _ = z           := h₂
-    divides _ (2*z) := divides_mul ..
+    Divides _ (2*z) := Divides_mul ..
 
-infix:50 " ∣ " => divides
-
-example (h₁ : divides x y) (h₂ : y = z) : divides x (2*z) :=
-  calc
-    x ∣ y   := h₁
-    _ = z   := h₂
-    _ ∣ 2*z := divides_mul ..
```

Comment:
This notation is now built-in, and they were conflicting. It didn't seem to be worth a longer aside in the text to save this, so I deleted it.